src/util/deploy/create-deploy.js

```diff
@@ -61,6 +61,11 @@ export default async function createDeploy(
       return createDeploy(output, now, contextName, paths, createArgs);
     }
 
+    if (error.code === 'not_found') {
+      // The first argument is the deployment id, which is not available here
+      return new ERRORS_TS.DeploymentNotFound('', contextName)
```

Comment:
@leo what do you think about this? Should the `DeploymentNotFound` constructor be updated so the deployment id is optional, or can we get the id of the deployment here?

Reply:
@juliangruber I would update it to make it optional, yep.